README.md

```diff
@@ -0,0 +1,16 @@
+# Hello Microverse
+
+> This is my first exercise in Microverse program.
+
+We learn how to use linters in our project.
+
+![Screenshot](./images/Capture.PNG)
+## Built With
+- Html
+- Css
+
+## Authors
+👤 **Beyk**
+- GitHub: [@beyk](https://github.com/beyk)
+- Twitter: [@beyk_a](https://twitter.com/beyk_a)
+- LinkedIn: [Beyk](https://www.linkedin.com/in/asghar-beykmohammadi-1b16b291/)
```

Comment:
- [ ] [OPTIONAL] I think it will be better to describe in a README file how to set up your project on an end-user machine. So, you should do it for future projects, that is why it is time to practice.

Reply:
Sure